x-pack/plugins/screenshotting/server/config/default_chromium_sandbox_disabled.ts

```diff
@@ -40,8 +40,15 @@ interface OsSummary {
 export async function getDefaultChromiumSandboxDisabled(): Promise<OsSummary> {
   const os = await getOs();
 
-  return {
-    os,
-    disableSandbox: os.os === 'linux' && !distroSupportsUnprivilegedUsernamespaces(os.dist),
-  };
+  let enableSandbox: boolean;
+  if (process.env.ELASTIC_CONTAINER) {
```

Comment:
can we also add a note in the Dockerfile how this is being used?

Reply:
837714d1fd50b057e34043ef9cb59576e75407fb